packages/derived-wallet-base/src/envelope.ts

```diff
@@ -16,8 +16,9 @@ function getChainName(chainId: number) {
       return network;
     }
   }
-  // Otherwise return a chain id descriptor
-  return `chainId: ${chainId}`;
+  // Otherwise return null. Should really only happen for devnet, and on chain
+  // the auth function does not expect a chain id for devnet
+  return null;
```

Comment:
Can we revert this and make the on-chain function match?

I don't see a reason against making this work for all chains, other than the fact that is currently not implemented in the on-chain authentication function at the moment.
I had left a comment in that PR not sure if I got a response on it, lemme check

Reply:
cc @BriungRi

Reply:
Ok [caught up](https://github.com/aptos-labs/aptos-core/pull/16236#discussion_r2027583199) in the other thread, whether we display something or nothing at all is up to y'all.

Just one nit, can we return `undefined` instead of `null`?